pkgs/development/python-modules/diagrams/default.nix

```diff
@@ -27,15 +27,19 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace 'jinja2 = "^2.10"' 'jinja2 = "*"'
+      --replace 'jinja2 = "^2.10"' 'jinja2 = "*"' \
+      --replace 'graphviz = ">=0.13.2,<0.17.0"' 'graphviz = "*"'
   '';
 
   preConfigure = ''
     patchShebangs autogen.sh
     ./autogen.sh
   '';
 
-  patches = [ ./build_poetry.patch ];
+  patches = [
+    # The build-system section is missing
```

Comment:
Thank you for noting why this patch is here!

Reply:
For humans it's clear but `nixpkgs-hammering` could complain :wink: